gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.4-bin.zip
```

Comment:
According to https://gradle.org/install/ the "_current Gradle release is 6.6.1_" -- why 6.4?

Reply:
> According to https://gradle.org/install/ the "_current Gradle release is 6.6.1_" -- why 6.4?

I only updated to the least needed version according to your other link to support modules. Didn't thought about going to a higher version  in hope to get less problems through the Gradle update in combination with trying to introduce the module file.

Reply:
Also it seems that upgrading to 6.6.1 needs further investigation, see #328 . So I'll stick to 6.4 for this PR.

Reply:
#328 was merged, so this has been resolved.